spells.yml

```diff
@@ -6408,14 +6817,15 @@ minor_illusion:
 
     If you create an image of an object—such as a chair, muddy footprints, or a small chest—it must be no larger than a 5-foot cube. The image can't create sound, light, smell, or any other sensory effect. Physical interaction with the image reveals it to be an illusion, because things can pass through it.
 
-    If a creature uses a standard action to examine the sound or image, the creature can determine that it is an illusion with a successful Intelligence (Investigation) check against your spell save DC. If a creature discerns the illusion for what it is, the illusion becomes faint to the creature.
+    A creature can use the Study action to inspect the sound or image, making an Investigation check against your spell save DC. On a success, the creature discovers that the sound or image is illusory. If a creature discerns the illusion for what it is, the illusion becomes faint to that creature.
```

Comment:
While we're modifying this, should we just use exactly the same language as with _major image_? I.e.:

> If a creature discerns the illusion for what it is, the creature can see through the image, and its other sensory qualities become faint to the creature.

I'm not sure what the right wording should be, but it seems odd that an Investigation check would let you see through _major image_ but not _minor illusion_.

Reply:
I ran into that when rewording them, and the catch is that minor illusion might just make a sound with nothing visible. I'll give it a shot though, because it should be clear that you can see through either on an Investigation